pkg/unvanquished_src.dpkdir/bots/build.bt

```diff
@@ -0,0 +1,78 @@
+selectClass
+{
+	selector
+	{
+		condition team == TEAM_ALIENS
+		{
+			spawnAs PCL_ALIEN_BUILDER0
+		}
+		condition team == TEAM_HUMANS
+		{
+			spawnAs WP_HBUILD
+		}
+	}
+}
+
+selector
+{
+	condition ( team == TEAM_HUMANS )
+	{
+		selector
+		{
+			behavior unstick
+			behavior use_medkit
+
+			sequence
+			{
+				condition !haveWeapon( WP_HBUILD )
+				selector
+				{
+					action buy( WP_HBUILD )
+					action suicide
+				}
+			}
+
+			decorator return ( STATUS_FAILURE )
+			{
+				action buildNowChosenBuildable
+			}
+
+			action roamInRadius( E_H_REACTOR, 700 )
+			action roam
+
+		}
+	}
+	condition ( team == TEAM_ALIENS )
+	{
+		selector
+		{
+			behavior unstick
+
+			sequence
+			{
+				condition class != PCL_ALIEN_BUILDER0 && class != PCL_ALIEN_BUILDER0_UPG
+				selector
+				{
+					sequence
+					{
+						action moveTo( E_A_OVERMIND )
+						selector
+						{
+							action evolveTo( PCL_ALIEN_BUILDER0_UPG )
+							action evolveTo( PCL_ALIEN_BUILDER0 )
+						}
+					}
+					action suicide
+				}
+			}
+
+			decorator return ( STATUS_FAILURE )
```

Comment:
I don't care one way or the other, but if you want to keep the behavior that the bot stops moving when there is nothing to build, you can use `decorator invert`